<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportFieldsDialog</class>
 <widget class="QDialog" name="ExportFieldsDialog">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkAllFields">
     <property name="text"><string>Tout s√©lectionner</string></property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listFields">
     <property name="selectionMode"><set>QAbstractItemView::MultiSelection</set></property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation"><enum>Qt::Horizontal</enum></property>
     <property name="standardButtons"><set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>